JavaScript/chapter02/p01_remove_dups/avc278.js

```diff
@@ -0,0 +1,101 @@
+// Remove Dups: Write code to remove duplicates from an unsorted linked list.
+// Follow Up: How would you solve this problem if a temporary buffer is not allowed?
+
+const assert = require("assert");
+
+class LinkedListNode {
+  constructor(val, next) {
+    this.val = val === undefined ? 0 : val;
+    this.next = next === undefined ? null : next;
+  }
+}
+
+/**
+ * Removes duplicates from a linked list
+ * @param  {LinkedList} LL input linked list to mutate
+ * @return {null}
+ *
+ * Keep a temporary set for storing values we pass through, of maximum size N where N is the number of nodes in the
+ * input linked list in the worst case. This requires an additional O(N) space. We store two pointers as we traverse
+ * through the linked list, skipping nodes with values already in our set.
+ * Runtime: O(N)
+ * Space:   O(N)
+ *
+ */
+const removeDups1 = (LL) => {
```

Comment:
per https://github.com/airbnb/javascript#naming--camelCase, the name `LL` might be confused with a class. how about `list`? same in L56.